addons/addon-base-raas/packages/base-raas-services/lib/schema/update-user-roles.json

```diff
@@ -10,22 +10,17 @@
       "pattern": "^[A-Za-z0-9-_ ]+$"
     },
     "rev": {
-      "type": "number"
+      "type": "number",
+      "minimum": 0
     },
     "description": {
       "type": "string",
       "maxLength": 2048
     },
     "userType": {
       "type": "string",
-      "enum": [
-        "INTERNAL", "EXTERNAL"
-      ]
+      "enum": ["INTERNAL", "EXTERNAL"]
     }
   },
-  "required": [
-    "id",
-    "rev",
-    "userType"
-  ]
+  "required": ["id", "rev", "userType"]
```

Comment:
Not sure if 'userType' should be required?

Reply:
It was there before just reformatted -- no material change